db/seed.rb

```diff
@@ -61,21 +62,25 @@ def self.trips
         zipcode: row[:zip_code],
         condition_id: Condition.find_by(date: Date.strptime(row[:start_date], '%m/%e/%Y')).id
       )
-
+    rescue
+      puts "Caught Seed Error"
```

Comment:
If we have to use a lot of these rescues (maybe where we are having errors in Station index) could we turn this into a method to call on?